support-frontend/app/admin/settings/Switches.scala

```diff
@@ -25,6 +25,7 @@ case class FeatureSwitches(
 case class CampaignSwitches(
     enableContributionsCampaign: Option[SwitchState],
     forceContributionsCampaign: Option[SwitchState],
+    usEoy2024: Option[SwitchState] = None,
```

Comment:
this needs to be added in the RRCP switchboard

Reply:
should we raise a card for that?

Reply:
it's actually done from the tool, and I've already added it in CODE + PROD!

Reply:
Oh cool!!